### PR TITLE
Improved task that updates deb repo

### PR DESCRIPTION
**Pull request by btoll**
## Summary

The operation that uses the `aptly` tool to create a new snapshot with the new package(s) and upload to the mirror on S3 is now a `mule` task.

## Test Plan

`STAGING=algorand-staging VERSION=2.2.0 CHANNEL=stable mule -f package-deploy.yaml package-deploy-deb`

### CODE REVIEW

File: package-deploy.yaml [hunk 1]
```diff
@@ -24,6 +24,24 @@ agents:
...
+      - GOLANG_VERSION=`./scripts/get_golang_version.sh`
+    env:
+      - AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+      - CHANNEL=$CHANNEL
+      - SNAPSHOT=$SNAPSHOT
+      - STAGING=$STAGING
+      - VERSION=$VERSION
+    volumes:
+      - $XDG_RUNTIME_DIR/gnupg/S.gpg-agent:/root/.gnupg/S.gpg-agent
```
Comment (onetechnical) [inline package-deploy.yaml:41]: Not to fix here, but this is probably one of those things we'll have to look at with Mac clients.

**Review by onetechnical — APPROVED** [review]
Everything here looks fine, though I had a couple questions/comments.

File: docker/build/aptly.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:18.04
+
+ARG ARCH=amd64
+ARG GOLANG_VERSION
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && apt-get install aptly awscli binutils build-essential curl gnupg2 silversearcher-ag tree -y
```
[the remark this answers is not in the document]
Comment (btoll) [reply docker/build/aptly.Dockerfile:7]: I'm going to remove the non-essential packages.

File: package-deploy.yaml (hunk 1 above)
Comment (btoll) [reply package-deploy.yaml:41]: Agree, it's on my list :)

**Review by bricerisingalgorand — APPROVED** [review]

**Review by onetechnical — APPROVED** [review]

**Pull request merged**